### ISSUE TITLE
Se connecter en tant que Parents via FranceConnect ?

### ISSUE THREAD

**Issue by marlenicoclem** [delphiki/hass-pronote#42]
Bonjour, 
Je viens d'intégrer le module et je souhaite me connecter en tant que "Parents" via mes identifiants "France Connect".
Est-ce possible?
Dans le champs ENT, que dois-je mettre? sachant que je souhaite me connecter sur un collège de l'académie de Toulouse.
Merci beaucoup par avance. :)

**Comment by lesissou** [comment]
Bonjour,
j'ai le même soucis et la documentation n'est pas claire à ce sujet
J'ai tenté plusieurs solutions mais sans succès

**Comment by Ekomak** [comment]
Bonjour 
Je rencontre la même chose avec des identifiants EDUCONNET. 
Erreur authentification même en sélectionnant l'ENT qui correspond à mon académie.
Si vous avez une piste ou besoin d'informations pour debugger cela, je suis disponible.

Merci

**Comment by vingerha** [comment]
Quelle lien? Le lien doit avoir le format https://0123456.index-education.net/pronote/
Sinon essair v0.10.0 avec qrcode, ça contourne les ENT

**Comment by lesissou** [comment]
Super Merci @vingerha 
j'ai réussi à me connecter

**Comment by marlenicoclem** [comment]
> Quelle lien? Le lien doit avoir le format https://0123456.index-education.net/pronote/ Sinon essair v0.10.0 avec qrcode, ça contourne les ENT

Bonjour,
Pourrais-tu me donner plus de précisions sur ce qu'est la v.0.10.0 stp?
Merci par avance :)

**Comment by vingerha** [comment]
> Bonjour, Pourrais-tu me donner plus de précisions sur ce qu'est la v.0.10.0 stp? Pour ce qui est l'adresse, je rentre bien la bonne :) Merci par avance :)
L'option pour se connecter par QR code était introduit, voir la documentation

**Comment by marlenicoclem** [comment]
> > L'option pour se connecter par QR code était introduit, voir la documentation

Effectivement, je viens de tester avec le QR CODE que je génère à partir de pronote.
Je sélectionne bien 
Ensuite j'analyse le qr code et j'ai bien mon jeton, mon login, et je récupère bien l'url de mon établissement
{"jeton":"xxx","login":"xxx","url":"https://xxx.index-education.net/pronote/mobile.parent.html"}
J'ai essayé aussi avec l'url https://xxx.index-education.net/pronote/ mais il m'affiche l'erreur "Unknown error occurred"
J'ai bien saisi le code pin que j'ai entré pour générer le qr code.
Je sélectionne bien 
type de connexion par QR CODE 
Type de compte PARENTS

Merci pour votre aide

**Comment by vingerha** [comment]
> Merci pour votre aide
il ne faut jamais changer l'url dans le truc-jeton entre les {} ... t'as quoi comme dans les logs?
Si pas encore fait...dans configuration.yaml il fau tavoir qq comme ça
```
logger:
  default: warning
  logs:
    custom_components.pronote: debug

**Comment by patrickgmail** [comment]
Bonsoir
Pour les académies sans ENT (la 29ème : les lycées français de l'étranger), le fait de laisser vide le champ ENT fonctionne pour les parents.
Ce qui ont des problèmes d'authentification doivent mal écrier l'url pourtant donnée partout :
https://xxx.index-education.net/pronote/
Cette URL se termine bien avec le "/".
